gutenberg-starter-theme-blocks/experimental-theme.json

```diff
@@ -0,0 +1,204 @@
+{
+	"global": {
+		"presets": {
+			"color": [
+				{
+					"slug": "background",
+					"value": "#fff"
+				},
+				{
+					"slug": "background-dark",
+					"value": "#e6e6e6"
+				},
+				{
+					"slug": "foreground",
+					"value": "#191e23"
+				},
+				{
+					"slug": "foreground-light",
+					"value": "#666"
+				},
+				{
+					"slug": "border",
+					"value": "#ccc"
+				},
+				{
+					"slug": "primary",
+					"value": "#0073aa"
+				},
+				{
+					"slug": "primary-light",
+					"value": "#229fd8"
+				},
+				{
+					"slug": "secondary",
+					"value": "#444"
+				},
+				{
+					"slug": "secondary-light",
+					"value": "#eee"
+				}
+			],
+			"font-size": [
+				{
+					"slug": "small",
+					"value": "13px"
+				},
+				{
+					"slug": "normal",
+					"value": "16px"
+				},
+				{
+					"slug": "medium",
+					"value": "20px"
+				},
+				{
+					"slug": "large",
+					"value": "36px"
+				},
+				{
+					"slug": "huge",
+					"value": "48px"
+				}
+			],
+			"line-height": [
+				{
+					"slug": "small",
+					"value": "1.2"
+				},
+				{
+					"slug": "medium",
+					"value": "1.4"
+				},
+				{
+					"slug": "large",
+					"value": "1.8"
+				}
+			]
+		},
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"text": "var( --wp--preset--color--foreground )"
+			}
+		}
+	},
+	"core/paragraph": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
```

Comment:
The result of setting this right now is that any paragraph will have a background color applied, e.g. inside the cover block:

<img width="1904" alt="Screen Shot 2020-07-21 at 10 15 09 AM" src="https://user-images.githubusercontent.com/5375500/88065841-1b50a380-cb3b-11ea-99b1-9e117b72ced4.png">

Reply:
Hmm interesting. That's not great. Should we not specify a value by default then?

Reply:
Could we set it to `inherit`?

Reply:
I think not specifying a color is what we want here — by default we want all those blocks to just be transparent. I've pushed an update to do that.